docker-compose.dev.yml

```diff
@@ -7,10 +7,8 @@ services:
     ports:
       - "5432:5432"
     restart: unless-stopped
-    environment:
-      - POSTGRES_PASSWORD=${DATABASE_PASSWORD}
-      - POSTGRES_USER=${DATABASE_USER}
-      - POSTGRES_DB=${DATABASE_NAME}
```

Comment:
I think we should keep it like this with the `environment:` key. If we change it like you proposed, we either duplicate environment variables (as you have done with e.g. `POSTGRES_USER` and `DATABASE_USER` having the same value), or we would need to change everything to the environment variables expected by Postgres (e.g. `POSTGRES_PASSWORD` instead of `DATABASE_PASSWORD`). This would introduce breaking changes for existing developments, and I don't think it's worth it in this case.

We can deal with the 3 env vars here. The same goes for the other two `docker-compose.*` files.